### PR TITLE
Modified readme and moved image to gcr.io repository. Issue #35

### PR DESCRIPTION
**Pull request by amsaha**

### CODE REVIEW

File: mnist/README.md [hunk 1]
```diff
@@ -48,42 +48,47 @@ logical GKE cluster where the `train` and `serve` stages run) connects with the
...
 
-   Point `DOCKER_BASE_URL` to your DockerHub account. Point `IMAGE` to your training image. If you don't have a DockerHub account, create one.
+   Point `DOCKER_BASE_URL` to your DockerHub account. Point `IMAGE` to your training image. If you don't have a DockerHub account,
+   create one at [https://hub.docker.com/](https://hub.docker.com/), upload your image there, and do the following
+   (replace <username> and <container> with appropriate values).
 
-       DOCKER_BASE_URL=docker.io/johnugeorge
-       IMAGE=${DOCKER_BASE_URL}/tfmodel
+       DOCKER_BASE_URL=docker.io/<username>
+       IMAGE=${DOCKER_BASE_URL}/<container>
```
Comment (johnugeorge) [inline mnist/README.md]: Instead of container, image-name?

Comment (amsaha) [reply mnist/README.md]: fixe.

**Pull request merged**